README.md

```diff
@@ -865,6 +854,89 @@ var timestamp = time.ZonedDateTime.now().plusMinutes(5);
 console.log(timestamp.getMillisFromNow());
 ```
 
+### Quantity
+
+The `Quantity` class greatly simplifies Units of Measurement (UoM)/Quantity handling by providing unit conversion, comparisons and mathematical operations.
+
+Internally using the openHAB `QuantityType`, which relies on [`javax.measure`](https://unitsofmeasurement.github.io/unit-api/), it supports all units and dimensions that openHAB supports - see [Units of Measurement documentation](https://www.openhab.org/docs/concepts/units-of-measurement.html#list-of-units) for an (incomplete) overview.
+If you unit is not listed in the UoM docs, it is very likely that it is still supported, e.g. the Angstrom Å for very small lengths (1 Å = 10 nm).
+
+#### Creation
+
+`Quantity(value)` can be used as a factory (without `new` - don't use it as a constructor), pass an amount **and** a unit to it to create a new `Quantity` object:
+
+
+```javascript
+// Allowed:
+var qty = Quantity('5.75 m');
+qty = Quantity('1 N*m');
+qty = Quantity('1 m/s');
+qty = Quantity('1 m^2/s^2');
+qty = Quantity('1 m^2/s^-2'); // negative powers
+qty = Quantity('1'); // unitless quantity
+
+// Not allowed:
+qty = Quantity('m');
+qty = Quantity('1 N m'); // * is required
+qty = Quantity('1 m^2 s^2'); // / is required
+qty = Quantity(1 m2/s2); // ^ is required
+```
+
+`value` can be a string, a `Quantity` instance or an openHAB Java [`QuantityType`](https://www.openhab.org/javadoc/latest/org/openhab/core/library/types/quantitytype).
+
+`value` strings have the `$amount $unit` format and must follow these rules:
+
+- `$amount` is required and a number as string
+- `$unit` is optional (unitless quantities are possible) and can have a prefix like `m` (milli) or `M` (mega)
+- If a `$unit` contains `*`, `/` and `^`, they are not optional and must be present.
+
+#### Conversion
+
+It is possible to convert a `Quantity` to another unit or get the value as integer or float:
+
+```javascript
+var qty = Quantity('10.2 °C');
+
+qty = qty.toUnit('°F');
+var intValue = qty.int;
+var floatValue = qty.float;
+```
+
+`toUnit` returns a new Quantity with the given unit or `null`, if conversion to that unit is not possible.
+
+#### Comparison
+
+`Quantity` provides the following methods for comparison:
+
+- `equal(value)` ⇒ `boolean`: this `Quantity` === another `Quantity`
+- `largerThan` ⇒ `boolean`: this `Quantity` > another `Quantity`
```

Comment:
We usually say "greaterThat" instead of "largerThan". Both make sense but "greater" is closer to how we are taught in elementary school. I'm not sure if that's just a US thing or English speaking world universal.

Reply:
https://www.w3schools.com/js/js_comparisons.asp is using greater and less, not larger and smaller.

I have no strong opinion here, I guess we can change this although it is released, but I don't expect many to use it as it was released a few hours ago and is undocumented.

Reply:
I'd leave it up to you. It does throw a little bit of head tilt when seeing it as larger instead of greater.

Reply:
I will check if I can change it.

Reply:
> We usually say "greaterThat" instead of "largerThan"

Really "greaterTha**t**"?

Reply:
I’d change it to greaterThan

This is how w3schools has it and the Mozilla WebDev Docs also have it with than, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators#relational_operators.

Reply:
> Really "greaterThat"?

No, that's a typo. It's greaterThan and lessThan, same as in the w3schools link @florian-h05 posted.

Reply:
@rkoshak 
Since the current version already has 60 downloads from npm, I’ll publish the changed method names together with the changes from #212 in a new major version to follow semver.